### PR TITLE
Translate Ruby 3.3.6 Released (zh_tw)

### PR DESCRIPTION
**Pull request by timfanda35**
Translate Ruby 3.3.6 Released (zh_tw)

Thank you.

### CODE REVIEW

File: zh_tw/news/_posts/2024-11-05-ruby-3-3-6-released.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+author: k0kubun
+translator: "Bear Su"
+date: 2024-11-05 04:25:00 +0000
+lang: zh_tw
+---
+
+Ruby 3.3.6 已經發布了。
+
+這是包括修復小型錯誤的例行更新。
+此版本會停止警告缺少預設 gems 的依賴項目，如果這些項目將在 Ruby 3.5 被 bundled。
```
Comment (JuanitoFatas) [inline zh_tw/news/_posts/2024-11-05-ruby-3-3-6-released.md:13]: Reads a bit unnatural 🤔 

此版本會停止警告缺少預設 gems 的依賴項目，如果這些項目將在 Ruby 3.5 被 bundled。

Comment (timfanda35) [reply zh_tw/news/_posts/2024-11-05-ruby-3-3-6-released.md:13]: > It also stops warning missing default gem dependencies that will be bundled gems in Ruby 3.5.

How about this version?

此版本也停止有關缺少預設 gem 依賴項目的警告，這些 gems 將在 Ruby 3.5 成為 bundled gems。

**Review by JuanitoFatas — APPROVED** [review]
🙌🏻

File: zh_tw/news/_posts/2024-11-05-ruby-3-3-6-released.md (hunk 1 above)
Comment (JuanitoFatas) [reply zh_tw/news/_posts/2024-11-05-ruby-3-3-6-released.md]: Sorry I was busy, LGTM now!!

**Pull request merged**